arm-network/2017-03-01/swagger/virtualNetwork.json

```diff
@@ -983,6 +1025,70 @@
         }
       },
       "description": "Response for CheckIPAddressAvailability API service call"
+    },
```

Comment:
i would also put this entire change into the usage.json file, if it makes things simpler